src/run_build.py

```diff
@@ -45,8 +48,16 @@ def main() -> int:
     if args.incremental:
         buildIncremental = BuildIncremental(manifest, args.distribution)
         list_of_updated_plugins = buildIncremental.commits_diff(manifest)
-        logging.info(f"Plugins for incremental build: {buildIncremental.rebuild_plugins(list_of_updated_plugins, manifest)}")
-        return 0
+        components = buildIncremental.rebuild_plugins(list_of_updated_plugins, manifest)
+        logging.info(f"Plugins for incremental build: {components}")
+
+        build_manifest_path = os.path.join(args.distribution, "builds", manifest.build.filename, "manifest.yml")
+        if not os.path.exists(build_manifest_path):
+            logging.error(f"Previous build manifest missing at path: {build_manifest_path}")
+
+        logging.info(f"Build {components} incrementally.")
+
+        build_manifest_data = BuildManifest.from_path(build_manifest_path).__to_dict__()
```

Comment:
Rename this to `build_manifest` and pass around as `BuildManifest`, rather than a dict.

Reply:
Thanks. Working on the changes.